### PR TITLE
Throw user exception if oauth_api version 2 is used + Fix for injected credentials

### PR DESCRIPTION
**Pull request by ujovlado**
Changes:

- Reverts keboola/docker-bundle#444 
- Fixes auth for Generic Ex (injected credentials)

### CODE REVIEW

File: src/Docker/Runner/Authorization.php [hunk 1]
```diff
@@ -45,26 +45,27 @@ public function getAuthorization($configData)
...
-                    $configData['oauth_api']['id']
-                );
-                $decrypted = $this->encryptor->decrypt($credentials);
-                $data['oauth_api']['credentials'] = $decrypted;
-            } catch (RequestException $e) {
-                if (($e->getCode() >= 400) && ($e->getCode() < 500)) {
-                    throw new UserException($e->getMessage(), $e);
-                } else {
-                    throw new ApplicationException($e->getMessage(), $e);
+            if (isset($configData['oauth_api']['version']) && ($configData['oauth_api']['version'] == 3)) {
```
Comment (odinuv) [inline src/Docker/Runner/Authorization.php]: zvladl bys pridat nekam semka https://github.com/keboola/docker-bundle/blob/master/Tests/Runner/AuthorizationTest.php#L183 kontrolu ze se getDetail nezavola ?

**Comment by ujovlado** [comment]
@odinuv preslo to, mozem este pridat test

File: src/Docker/Runner/Authorization.php (hunk 1 above)
Comment (ujovlado) [reply src/Docker/Runner/Authorization.php]: Myslim, ze ano.

Comment (ujovlado) [reply src/Docker/Runner/Authorization.php]: Je to tam

**Comment by ujovlado** [comment]
Myslim, ze ano :)

**Pull request merged**